trestle/tasks/cis_xlsx_to_oscal_cd.py

```diff
@@ -0,0 +1,887 @@
+        logger.info(text1 + text2)
+        text1 = '  profile-source             = '
+        text2 = f'(required) profile source, e.g., "{self._example_profile_source}".'
+        logger.info(text1 + text2)
+        text1 = '  profile-description        = '
+        text2 = f'(required) profile description, e.g., "{self._example_profile_description}".'
+        logger.info(text1 + text2)
+
+    def simulate(self) -> TaskOutcome:
+        """Provide a simulated outcome."""
+        return TaskOutcome('simulated-success')
+
+    def execute(self) -> TaskOutcome:
+        """Provide an actual outcome."""
+        try:
+            return self._execute()
+        except Exception:
+            logger.info(traceback.format_exc())
+            return TaskOutcome('failure')
+
+    def _execute(self) -> TaskOutcome:
+        """Wrap the execute for exception handling."""
+        if not self.config_object:
+            logger.warning('config missing')
+            return TaskOutcome('failure')
+        # required
+        try:
+            self._benchmark_file = self.config_object['benchmark-file']
+            self._oscal_cd_dir = self.config_object['output-dir']
+            self._namespace = self.config_object['namespace']
+            self._profile_version = self.config_object['profile-version']
+        except KeyError as e:
+            logger.info(f'key {e.args[0]} missing')
+            return TaskOutcome('failure')
+        # output
+        self._oscal_cd_path = pathlib.Path(self._oscal_cd_dir)
+        # insure output dir exists
+        self._oscal_cd_path.mkdir(exist_ok=True, parents=True)
+        # calculate output file name & check writability
+        oname = 'component-definition.json'
+        ofile = self._oscal_cd_path / oname
+        overwrite = self.config_object.get('output-overwrite', self._default_output_overwrite)
+        if not overwrite and pathlib.Path(ofile).exists():
+            logger.warning(f'output: {ofile} already exists')
+            return TaskOutcome('failure')
+        with self._get_tempdir() as tmpdir:
+            # step 1 - add combined sheet, if needed
+            combine_helper = CombineHelper(self.config_object, tmpdir)
+            combine_helper.run()
+            # step 2 - create trestle ready csv file from xlsx file
+            xlsx_to_csv_helper = XlsxToCsvHelper(self.config_object, tmpdir)
+            xlsx_to_csv_helper.run()
+            # step 3 - create OSCAL json file from csv file
+            csv_to_json_helper = CsvToJsonHelper(self.config_object, tmpdir)
+            task_outcome = csv_to_json_helper.run()
+            return task_outcome
+        # failure
+        logger.info(traceback.format_exc())
+        return TaskOutcome('failure')
+
+    def _get_tempdir(self) -> tempfile.TemporaryDirectory():
+        """Get tmpdir."""
+        return tempfile.TemporaryDirectory()
+
+
+@staticmethod
+def compare(item1, item2):
+    """Compare."""
```

Comment:
Removing "@staticmethod" annotation to compare function solved the issue on python 3.9

Reply:
Thx! Will give it a try...